scripts/test_reports/README.md

```diff
@@ -36,6 +36,17 @@ optional arguments:
                         Milestone ID
   -P PLAN, --plan PLAN  Test plan ID
 
+  -b buginfo_file, --buginfo-file buginfo_file for sanitycheck & sanitycheckbatch runner
```

Comment:
You have some misprints, and difficult to understand text. That change can solve it.
`  -b buginfo_file, --buginfo-file buginfo_file for sanitycheck & sanitycheckbatch runners
                     points to a file containing bug info for the error/failure cases.
                     When test cases results will be uploaded into the Test Rail, the bug IDs will be auto added for the error/failure cases.
                     The file's format is following:`

Reply:
@maksimmasalski  I have updated the README file ,and also submitted it again, please help to review it again. Thanks.